docker/labkey-20.3-community/README.md

```diff
@@ -0,0 +1,48 @@
+samples/docker/labkey-20.3-community
+==========
+
+This folder contains a sample docker-compose file and sample Docker configurations to stand up an instance of LabKey Server Community edition for testing.
+
+### IMPORTANT: Not Recommended for Production use
+ This image was meant for trying out LabKey Server and not meant for running a production server or for storing important biomedical data. 
```

Comment:
"This image is meant for trying out LabKey Server and not meant for running LabKey as a production service."

Reply:
fixed